raw-data/UW/uw_inventories.tsv

```diff
@@ -192,3 +192,176 @@ heath2016donnoso.pdf	donn1239	Donno So Dogon	Wendegele	pʰ	"Voiceless stops occu
+				eː		eː		
+						ẽː	"a nasal... or nasalized sonorant... at the onset of a syllable... induces nasalization on an intervocalic oral sonorant... at the beginning of the following syllable. Phonetically... the nasalization extends to adjoining vowels"	
+				ɛ	-ATR	ɛ		
+						ɛ̃	"a nasal... or nasalized sonorant... at the onset of a syllable... induces nasalization on an intervocalic oral sonorant... at the beginning of the following syllable. Phonetically... the nasalization extends to adjoining vowels"	
+				ɛː		ɛː		
+						ɛ̃ː	"a nasal... or nasalized sonorant... at the onset of a syllable... induces nasalization on an intervocalic oral sonorant... at the beginning of the following syllable. Phonetically... the nasalization extends to adjoining vowels"	
+				ɛ̃ː	"Leaving aside loanwords, affixal contractions, and an occasional expressive reduplication... nasalization is confined to long-vowel monosyllables, plus a few expressive adverbials"	ɛ̃ː		
+				a		a		
+						ã	"a nasal... or nasalized sonorant... at the onset of a syllable... induces nasalization on an intervocalic oral sonorant... at the beginning of the following syllable. Phonetically... the nasalization extends to adjoining vowels"	
+				aː		aː		
+						ãː	"a nasal... or nasalized sonorant... at the onset of a syllable... induces nasalization on an intervocalic oral sonorant... at the beginning of the following syllable. Phonetically... the nasalization extends to adjoining vowels"	
+				ãː	"Leaving aside loanwords, affixal contractions, and an occasional expressive reduplication... nasalization is confined to long-vowel monosyllables, plus a few expressive adverbials"	ãː		
+				u		u		
+						ũ	"a nasal... or nasalized sonorant... at the onset of a syllable... induces nasalization on an intervocalic oral sonorant... at the beginning of the following syllable. Phonetically... the nasalization extends to adjoining vowels"	
+				uː		uː		
+						ũː	"a nasal... or nasalized sonorant... at the onset of a syllable... induces nasalization on an intervocalic oral sonorant... at the beginning of the following syllable. Phonetically... the nasalization extends to adjoining vowels"	
+				o	+ATR	o		
+						õ	"a nasal... or nasalized sonorant... at the onset of a syllable... induces nasalization on an intervocalic oral sonorant... at the beginning of the following syllable. Phonetically... the nasalization extends to adjoining vowels"	
+				oː		oː		
+						õː	"a nasal... or nasalized sonorant... at the onset of a syllable... induces nasalization on an intervocalic oral sonorant... at the beginning of the following syllable. Phonetically... the nasalization extends to adjoining vowels"	
+				ɔ	-ATR	ɔ		
+						ɔ̃	"a nasal... or nasalized sonorant... at the onset of a syllable... induces nasalization on an intervocalic oral sonorant... at the beginning of the following syllable. Phonetically... the nasalization extends to adjoining vowels"	
+				ɔː		ɔː		
```

Comment:
When I go to this file, GitHub tells me that there is "illegal quoting" in line 361 (so it can't make the file "beautiful and searchable"), but I'm not sure what the issue is because there are no quotation marks in this line.

Reply:
The problem is probably [line 363](https://github.com/phoible/dev/blob/0f42b73f14effcd2765b27b433057a4b853ee2aa/raw-data/UW/uw_inventories.tsv#L363) actually.  Sometimes GitHub forgets how to count when things like this happen